src/Sylius/Bundle/ReviewBundle/Resources/config/services.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" ?>
+
+<!--
+
+ This file is part of the Sylius package.
+
+ (c) Paweł Jędrzejewski
+
+ For the full copyright and license information, please view the LICENSE
+ file that was distributed with this source code.
+
+-->
+
+<container xmlns="http://symfony.com/schema/dic/services"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+    <services>
+        <!--Subscribers-->
+        <service id="sylius.event_subscriber.review.load_metadata" class="Sylius\Bundle\ReviewBundle\EventListener\LoadMetadataSubscriber">
+            <argument>%sylius.review.subjects%</argument>
+            <tag name="doctrine.event_subscriber" />
+        </service>
+
+        <!--Validators-->
+        <service id="sylius.review.validator.unique_user_email" class="Sylius\Bundle\ReviewBundle\Validator\Constraints\UniqueUserEmailValidator">
```

Comment:
I am not sure we should prefix it with a ``review``, I generally suggest avoiding grouping services/parameters according to bundles, because it does not make much sense. We had that long time ago and it was annoying.